securedrop/bin/dev-shell

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# This script runs an arbitrary command in a containerized dev environment.
+set -euo pipefail
+most_recent_tag=$(docker images -q securedrop-test |head -n1)
+args="$*"
+if (( $# == 0 )); then
+  args=("bash")
+fi
+docker_args=()
+if [ -z "${SKIP_PORTS:-}" ]; then
+  docker_args+=(
+    -p8080:8080 -p8081:8081 -p5901:5901
+  )
+fi 
+docker_args+=(
+  -v "$(pwd):/app"
```

Comment:
What about -v "$(pwd):$(pwd)" here ? The problem with changing to /app is that the .pyc created in the container won't be useable outside of the container and vice versa.